olive/passes/onnx/optimum_conversion.py

```diff
@@ -42,6 +52,14 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
                 default_value=True,
                 description="Whether to skip post-processing the exported model.",
             ),
+            "legacy": PassConfigParam(
```

Comment:
In my PR for mistral, I pass the legacy in extra_args. So, one question is should we expose the legacy explicitly? 

One more thing I hit when testing is the no_post_process is related to legacy. When I set legacy=True and no_post_process=False, the exception will be raised. you can see https://github.com/microsoft/Olive/blob/3037a4f8cf51f6a559279f8bd20a81a5bcae0f98/olive/passes/onnx/optimum_conversion.py#L95 for more info. 

On anther hand, it seems the default value for no_post_process is True, but the its default value is False in optimum. Need we make them consistent?

Reply:
If we don't expose the `legacy` option, the pass would behave differently for optimum 1.14.0+ and before when the user doesn't provide the legacy value themselves. 

For the `no_post_process` and `legacy` incompatibility, we will need to look into this more. maybe in the source code.

Reply:
Moved to `extra_args` as discussed offline.